content/docs/ops/setup/app-health-check/index.md

```diff
@@ -27,27 +28,7 @@ This task provides examples for the first two options with Istio mutual TLS enab
 
 ## Liveness and readiness probes with command option
 
-In this section, you configure health checking when mutual TLS is disabled, then when mutual TLS is enabled.
-
-### Mutual TLS disabled
-
-Run this command to deploy [liveness]({{< github_file >}}/samples/health-check/liveness-command.yaml) in the default namespace:
-
-{{< text bash >}}
-$ kubectl apply -f <(istioctl kube-inject -f @samples/health-check/liveness-command.yaml@)
-{{< /text >}}
-
-Wait for a minute and check the pod status:
-
-{{< text bash >}}
-$ kubectl get pod
-NAME                             READY     STATUS    RESTARTS   AGE
-liveness-6857c8775f-zdv9r        2/2       Running   0           1m
-{{< /text >}}
-
-The number '0' in the 'RESTARTS' column means liveness probes worked fine. Readiness probes work in the same way and you can modify `liveness-command.yaml` accordingly to try it yourself.
-
-### Mutual TLS enabled
+In this section, you first configure health checking when mutual TLS is enabled.
```

Comment:
I fixed the sentence but the text makes no logical sense to readers. How they can do this is not discussed. Please revise.
```suggestion
First, you need to configure health checking with mutual TLS enabled.
```